Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -170,6 +170,9 @@ - (void)setup {
     self.gregorian = [[NSCalendar alloc] initWithCalendarIdentifier:NSCalendarIdentifierGregorian];
 
     self.calendarView = [[FSCalendar alloc] initWithFrame:CGRectZero];
+    if (self.configuration.rowHeight > 0) {
+        self.calendarView.rowHeight = self.configuration.rowHeight;
+    }
```

Comment:
We need to make this configurable so that the price label config can specify a taller cell size.